src/platform-includes/enriching-events/breadcrumbs/automatic-breadcrumbs/react-native.mdx

```diff
@@ -0,0 +1,91 @@
+The React Native SDK already captures breadcrumbs automatically via the Native SDKs for Android and iOS.
+
+- [Automatic Breadcrumbs for Android](/platforms/android/enriching-events/breadcrumbs/#automatic-breadcrumbs)
+- [Automatic Breadcrumbs for iOS](/platforms/apple/configuration/integrations/default/#sentryautobreadcrumbtrackingintegration)
+
+- For interaction breadcrumbs wrapp your `App` using the `Sentry.wrap` method.
+
+```javascript
+import * as React from 'react';
+import { Text, View } from 'react-native';
+import * as Sentry from '@sentry/react-native';
+
+Sentry.init({
+  // parameters
+});
+
+const App = () => {
+  return (
+    <View>
+      <Text>Hello World</Text>
+    </View>
+  );
+};
+
+export default Sentry.wrap(App);
+```
+
+- For navigation breadcrumbs use one of the SDK's natigation integrations.
+    - `Sentry.ReactNativeNavigationInstrumentation`
+    - `Sentry.ReactNavigationInstrumentation`
+    - `Sentry.ReactNavigationV4Instrumentation`
+
+```javascript
+import * as React from 'react';
+import { Text, View } from 'react-native';
+import {
+  NavigationContainer,
+  NavigationContainerRef,
+} from '@react-navigation/native';
+import * as Sentry from '@sentry/react-native';
+
+const instrumentation = new Sentry.ReactNavigationInstrumentation();
+
+Sentry.init({
+  // Other parameters
+  integrations: [
+    new Sentry.ReactNativeTracing({
+      routingInstrumentation: instrumentation,
+    }),
+  ],
+});
+
+const App = () => {
+  const navigation = React.useRef<NavigationContainerRef>();
+
+  return (
+    <NavigationContainer
+      ref={navigation}
+      onReady={() =>
+        instrumentation.registerNavigationContainer(navigation)}>
+      {/* Navigator and screens */}
+    </NavigationContainer>
+  );
+};
+```
+
+- To track `HTTP` requests enable tracing either by `tracesSampler` or `tracesSampleRate` and `enableAutoPerformanceTracking` options. Or setup your own `Sentry.RectNativeTracing`.
```

Comment:
I don't think we need that for breadcrumbs.
See https://docs.sentry.io/platforms/react-native/enriching-events/breadcrumbs/#automatic-breadcrumbs
This is enabled by default and independent of tracing.

Reply:
Ill check that because I looked it to the code and it doesn’t look enabled by default.

Reply:
Okay, it does work like a charm.
I confused it with tracing xhr.
https://github.com/getsentry/sentry-react-native/blob/7a58171df3ba20d4efe41620a24767445493d6c9/src/js/tracing/reactnativetracing.ts#L223